charts/merlin/values.yaml

```diff
@@ -26,167 +26,241 @@ deployment:
-  alertsRepoPlatform: gitlab
-  baseURL: https://gitlab.com/
-  # Access token must be specified using --set flag.
-  # token:
-  dashboardRepository: data-science/slo-specs
-  dashboardBranch: master
-  alertRepository: lens/artillery/datascience
-  alertBranch: master
-  warden:
-    apiHost: ""
-monitoring:
-  enabled: false
-  # baseURL: ""
-  # jobBaseURL: ""
-transformer:
-  simulation:
-    feastRedisServingURL: online-serving-redis.feast.dev
-    feastBigtableServingURL: online-serving-bt.feast.dev
-  feast:
-    defaultServingURL: online-serving-redis.feast.dev
-    servingURLs:
-      [
-        {
-          host: "online-serving-redis.feast.dev",
-          label: "Online Serving with Redis",
-          source_type: "REDIS",
-          icon: "redis",
-        },
-        {
-          host: "online-serving-bigtable.feast.dev",
-          label: "Online Serving with BigTable",
-          source_type: "BIGTABLE",
-          icon: "bigtable",
-        },
-      ]
-    ## Redis storage configuration for feast retrieval
-    ##
-    # redisStorage:
-    #   is_using_direct_storage: false
-    #   is_redis_cluster: true
-    #   serving_url: "online-serving-redis.feast.dev"
-    #   redis_addresses:
-    #     - 10.1.1.2
-    #     - 10.1.1.3
-    #   pool_size: 5
-    #   max_retries: 0
-    #   read_timeout: "1s"
-    #   write_timeout: "1s"
-    #   min_idle_conn: 2
-
-    ## Bigtable storage configuration for feast retrieval
-    ##
-    # bigtableStorage:
-    #   serving_url: "online-serving-bigtable.feast.dev"
-    #   is_using_direct_storage: false
-    #   project: "gcp-project"
-    #   instance: instance
-    #   app_profile: default
-    #   pool_size: 5
-    #   keep_alive_interval: "2m"
-    #   keep_alive_timeout: "1m"
-    defaultFeastSource: BIGTABLE
-    bigtableCredential:
-    coreURL: core.feast.dev
-    coreAuthAudience: core.feast.dev
-    authEnabled: false
-    grpc:
-      keepAliveEnabled: false
-      keepAliveTime: 60s
-      keepAliveTimeout: 5s
-  image: merlin-transformer:1.0.0
-  jaeger:
-    agentHost: localhost
-    agentPort: 6831
-    samplerType: const
-    samplerParam: 1
-    disabled: false
-  model:
-    grpc:
-      keepAliveEnabled: false
-      keepAliveTime: 60s
-      keepAliveTimeout: 5s
-  kafka:
-    brokers: "kafka-brokers"
-    maxMessageSize: "1048588"
+  k8s_config: {}
+  # Example k8s_config to connect to cluster using gke-gcloud-auth-plugin
+  # name: dev-cluster
+  # cluster:
+  #   server: https://k8s.cluster
+  #   certificate-authority-data: some_cert_data
+  # user:
+  #   exec:
+  #     apiVersion: client.authentication.k8s.io/v1beta1
+  #     args: ["--use_application_default_credentials"]
+  #     command: gke-gcloud-auth-plugin
+  #     interactiveMode: IfAvailable
+  #     provideClusterInfo: true
 
-# Google service account used to access GCP's resources.
-#
-# gcpServiceAccount:
-#   secretName: merlin-secret
-#   secretKey: service-account.json
```

Comment:
Removed this as it doesn't seem like it's used anymore (there's another `serviceAccount` block that does the same thing). Turing does not have this block either.